### PR TITLE
fix: revert success flag from trigger response

### LINKED ISSUE #15540: [Bug]: Appsmith add an additional success:true field to API response when it is triggered through an event
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

When the API is run through a button click, for example, there is a `"success" : true` field that is added to the API response. This is a problem when the user uses the JSON form widget as this field gets added as a switch widget on the form. 
unknown

Slack thread [link](https://theappsmith.slack.com/archives/C0341RERY4R/p1659078321039199)

### Steps To Reproduce

1. Create an API
2. Try running it through a button click
3. Check the response for "success" : true field.

### Public Sample App

_No response_

### Version

1.7.10

### PR DESCRIPTION
**Pull request by techbhavin**
## Description

> fix issue of showing additional property  success: true field to API response when it is triggered through an event

Fixes #15540 

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by sbalaji1192** [comment]
@techbhavin Added @Rishabh-Rathod as the reviewer.

**Comment by techbhavin** [comment]
this issue is happening because below change https://github.com/appsmithorg/appsmith/issues/3613#issuecomment-1138453150


For now, we revert a current change for now. 
@sbalaji1192  @Rishabh-Rathod  How we can add new changes mentioned in (https://github.com/appsmithorg/appsmith/issues/3613#issuecomment-1138453150) 

cc:  @somangshu @dilippitchika

File: app/client/src/sagas/ActionExecution/ActionExecutionSagas.ts [hunk 1]
```diff
@@ -188,16 +182,9 @@ function* initiateActionTriggerExecution(
   // it will be created again while execution
   AppsmithConsole.deleteError(`${source?.id}-${triggerPropertyName}`);
   try {
-    const res: unknown[] = yield call(executeAppAction, action.payload);
+    yield call(executeAppAction, action.payload);
```
Comment (Rishabh-Rathod) [inline app/client/src/sagas/ActionExecution/ActionExecutionSagas.ts:185]: To get the success value, What if the return value of `executeAppAction` is made as 
```js
{
   response, 
   success // true or false value
}
```
Will that solve the problem?

**Review by Rishabh-Rathod — APPROVED** [review]
Revert code LGTM.

**Comment by chandannkumar** [comment]
- Tested API run on widget Events trigger
- Tested API run from JS object

Issue of showing additional property success: true field to API response when it is triggered through an event or JS object is fixed

File: app/client/src/sagas/ActionExecution/ActionExecutionSagas.ts (hunk 1 above)
Comment (techbhavin) [reply app/client/src/sagas/ActionExecution/ActionExecutionSagas.ts:185]: @Rishabh-Rathod  can fix the problem in another issue for this change

Comment (Rishabh-Rathod) [reply app/client/src/sagas/ActionExecution/ActionExecutionSagas.ts:185]: Yes, we can fix the issue in another PR as this is critical issue.

**Pull request merged**